### DISCUSSION TITLE
Only colorize project / context, not task text

### DISCUSSION THREAD

**Discussion by MicAndHas** [todotxt/todo.txt-cli#423] [Q&A] [answered]
Hey all, I just have a quick question. I recently stumbled across this tool and I love it. However, I would love to only apply the priority highlighting to the task's project / context, not the task text itself. I tried to explore the `.cfg`  file and the `.sh` file to see if it was an option or easily hackable with my limited knowledge of bash scripting, with no avail. I just think that the full coloring of the task gets a little distracting and it would be nice to keep the task text at the default text color. Any help / response is much appreciated.

**Comment by inkarkat** [comment, marked as answer]
Sorry, that's not easily possible. It's not impossible, either; you'd have to clear all the `PRI_?` default colors, and then implement the highlighting (e.g. with `sed` or `awk`) yourself, and configure that filter as either `pre_filter_command`, `post_filter_command`, or `TODOTXT_FINAL_FILTER` (depending on where in the rendering pipeline it should run). But if you want to keep the default highlighting of project and contexts, getting the overlay of colors surely isn't trivial.

I'd rather spend some time on customizing the default colors to your liking. Modern terminals all support 256 or even true colors, so (depending on your tastes and chosen terminal colors), there's a lot of room for customization of the very conservative 16 colors palette. Lightly tinted background colors (similar to text markers) should be very readable yet not too distracting.
